src/StreetNameRegistry.Projections.Integration/StreetNameLatestItem.cs

```diff
@@ -0,0 +1,91 @@
+namespace StreetNameRegistry.Projections.Integration
+{
+    using System;
+    using Be.Vlaanderen.Basisregisters.GrAr.Common;
+    using Be.Vlaanderen.Basisregisters.Utilities;
+    using Microsoft.EntityFrameworkCore;
+    using Microsoft.EntityFrameworkCore.Metadata.Builders;
+    using NodaTime;
+    using StreetNameRegistry.Infrastructure;
+
+    public sealed class StreetNameLatestItem
+    {
+        public const string VersionTimestampBackingPropertyName = nameof(VersionTimestampAsDateTimeOffset);
+
+        public int PersistentLocalId { get; set; }
+
+        public string? Status { get; set; }
+
+        public string? NisCode { get; set; }
+
+        public string? NameDutch { get; set; }
+        public string? NameFrench { get; set; }
+        public string? NameGerman { get; set; }
+        public string? NameEnglish { get; set; }
+
+        public string? HomonymAdditionDutch { get; set; }
+        public string? HomonymAdditionFrench { get; set; }
+        public string? HomonymAdditionGerman { get; set; }
+        public string? HomonymAdditionEnglish { get; set; }
+
+        public bool IsRemoved { get; set; }
+
+        public string PuriId { get; set; }
+        public string Namespace { get; set; }
+        public string VersionAsString { get; set; }
+        private DateTimeOffset VersionTimestampAsDateTimeOffset { get; set; }
+
+        public Instant VersionTimestamp
+        {
+            get => Instant.FromDateTimeOffset(VersionTimestampAsDateTimeOffset);
+            set
+            {
+                VersionTimestampAsDateTimeOffset = value.ToDateTimeOffset();
+                VersionAsString = new Rfc3339SerializableDateTimeOffset(value.ToBelgianDateTimeOffset()).ToString();
+            }
+        }
+
+        public StreetNameLatestItem()
```

Comment:
Missing property IdempotencyKey? See https://github.com/Informatievlaanderen/municipality-registry/pull/578/files#diff-bbc7dece153b9ac64a5e8f00851a5c7d1bef1de0bc8e3afeac569fb3f4abbf5fR51

Reply:
@ArneD decided it could be removed because of ProjectionState position